### PR TITLE
Update badge

### PR DESCRIPTION
**Pull request by boygirl**
This PR updates the runpkg badge to use new designs. Also updates the favicon and social share images

Screen Shot 2020-03-16 at 1 09 30 PM

### CODE REVIEW

File: components/RunpkgIcon.js [hunk 1]
```diff
@@ -1,70 +1,52 @@
 import { html } from '../utils/rplus.js';
 
 export default html`
-  <svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 380 377">
-    <g filter="url(#filter0_d)">
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="600"
```
Comment (lukejacksonn) [inline components/RunpkgIcon.js]: This fixed width and height breaks the layout currently (pushes the instructional text to the bottom of the page). I think if you remove it and add check the spacing between the paragraph below it.. then we are good to go!

image

**Comment by lukejacksonn** [comment]
Ooo.. pretty! I just left one comment otherwise looks great 👍

**Comment by boygirl** [comment]
@lukejacksonn Thanks for catching that!

**Comment by boygirl** [comment]
@lukejacksonn not sure why some of the checks are hanging on my last commit. Is this fine to merge anyway?

**Comment by lukejacksonn** [comment]
Looking good! I don't know why those checks are hanging 🤷‍♂ but the preview deploy seems to work fine so lets just go for it :shipit:

**Pull request merged**